### PR TITLE
ci: add next tag

### PR DESCRIPTION
**Pull request by helciofranco**
- Publish `next` tag
- Add `unpublish` script

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
         id: changesets
         uses: FuelLabs/changesets-action@main
         with:
-          publish: pnpm changeset:release
+          publish: pnpm changeset publish --tag next
```
Comment (luizstacio) [inline .github/workflows/release.yaml:48]: On fuels-npm-packs we don't have pre-release next / rc. We should update the tag `main` to `next`. And keep this one as release latest.

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (helciofranco) [reply .github/workflows/release.yaml]: https://github.com/FuelLabs/fuels-npm-packs/pull/61/commits/ae973fe5ac3dc82c8190e6c30e6aca231765e021

File: scripts/unpublish.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+const { readFile, readdir } = require('node:fs').promises;
+const { join } = require('node:path');
+const util = require('node:util');
+const { compare } = require('compare-versions');
```
Comment (luizstacio) [inline scripts/unpublish.js:4]: You need to include this pacakge also here?

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: scripts/unpublish.js (hunk 2 above)
Comment (helciofranco) [reply scripts/unpublish.js:4]: nice catch
https://github.com/FuelLabs/fuels-npm-packs/pull/61/commits/b2d1725894e0a9f7ff3f12fd0d7f1e42a9f0ebb6

**Review by luizstacio — APPROVED** [review]

**Pull request merged**